InSimDotNet/PacketReader.cs

```diff
@@ -103,6 +112,34 @@ public class PacketReader {
             return BitConverter.ToSingle(buffer, position - 4);
         }
 
+        /// <summary>
+        /// Reads a LFS Car name or modded car skinID (as of 0.6W)
+        /// </summary>
+        /// <param name="count">The number of bytes to read.</param>
+        /// <returns>A Unicode string.</returns>
+        public string ReadCNameString(int count = 4)
+        {
+            position += count;
+
+            var buf = new byte[4];

```

Comment:
Just noticed, should this be initialised to the size of count?

Reply:
I added it as a default parameter just for readability if anyone editing this source (or the maintainer (you)) would want to add (4) when calling this method when parsing packet bytes